debugger/src/test/kotlin/org/rust/debugger/lang/RsFrameTypeDecoratorTest.kt

```diff
@@ -0,0 +1,197 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.lang
+
+import junit.framework.TestCase
+
+class RsFrameTypeDecoratorTest : TestCase() {
```

Comment:
Actually, I meant not to use `TestCase` at all, and use `@Test` annotation and `org.junit.Assert` for assertions

It's more a matter of taste, currently, but someday we will migrate all to JUnit 4/5 and it would be nice not to introduce new JUnit 3 tests if it's possible.

Reply:
Done